mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping

```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/class_4790 net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder
-	FIELD field_22204 GRAVEL Lnet/minecraft/class_2680;
-	FIELD field_22209 SOUL_SAND Lnet/minecraft/class_2680;
+	FIELD field_22204 VALLEY_GRAVEL Lnet/minecraft/class_2680;
+	FIELD field_22209 VALLEY_SOUL_SAND Lnet/minecraft/class_2680;
```

Comment:
```suggestion
	FIELD field_22204 GRAVEL Lnet/minecraft/class_2680;
	FIELD field_22209 SOUL_SAND Lnet/minecraft/class_2680;
```

Reply:
`GRAVEL` and `SOUL_SAND` already exist in a superclass as final, thus these need unique names.

Reply:
hmm, java allows you to declare static final fields/methods with same names in classes, and if super class fields are private you can safely rename them manually out of enigma